src/OrbitGl/App.cpp

```diff
@@ -676,9 +676,18 @@ void OrbitApp::OnErrorsWithPerfEventOpenEvent(
 
 void OrbitApp::OnWarningInstrumentingWithUprobesEvent(
     orbit_grpc_protos::WarningInstrumentingWithUprobesEvent
-    /*warning_instrumenting_with_uprobes_event*/) {
-  // TODO(b/232072696): Display WarningInstrumentingWithUprobesEvent in the Capture Log.
-  ORBIT_UNREACHABLE();
+        warning_instrumenting_with_uprobes_event) {
+  main_thread_executor_->Schedule([this, warning_instrumenting_with_uprobes_event = std::move(
+                                             warning_instrumenting_with_uprobes_event)]() {
+    std::string message = "Uprobes likely failed to instrument some functions:\n";
+    for (const auto& function :
+         warning_instrumenting_with_uprobes_event.functions_that_failed_to_instrument()) {
+      absl::StrAppend(&message, "* ", function.error_message(), "\n");
```

Comment:
Not that it actually matters, but I usually solve the `\n` with `absl::StrJoin`, to avoid a `\n` at the very end. (Also, naybe you added the new line at the end on purpose). You could do something like:
```
absl::StrJoin(functions, "\n", [](std::string* out, const auto& function) {
    absl::StrAppend(out, "* ", function.error_message());
});
```

Reply:
Yeah, the empty line was on purpose to match the empty line at the end for `WarningInstrumentingWithUserSpaceInstrumentationEvent` (and now, to match the empty line between the list of uninstrumented functions and the suggestion below).
@danielfenner Should we remove these empty lines?

Reply:
@danielfenner Any opinion on this? Personally I would leave the empty lines for now given that the messages can be long.

Reply:
I agree that an empty line between the functions and the message "Consider Orbit method" makes sense, for the reasons you mentioned. 

I would leave it as is